src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -2149,6 +2171,269 @@ def get_network_policy(self, enable_validation: bool = False, **kwargs) -> Union
 
         return network_policy
 
+    # pylint: disable=unused-argument
+    def get_enable_addons(self, **kwargs) -> Union[str, List[str], None]:
+        """Obtain the value of enable_addons.
+
+        Note: enable_addons will not be decorated into the `mc` object.
+
+        This function supports the option of enable_split. When enabled, it will split the string into a list with
+        "," as the delimiter.
+
+        :return: string, empty list or list of strings, or None
+        """
+        # read the original value passed by the command
+        enable_addons = self.raw_param.get("enable_addons")
+
+        if kwargs.get("enable_split", False):
```

Comment:
just curious, what's enable_split

Reply:
When this option is specified, we will split the string by comma. This parameter name is a little bit confusing, changed to `enable_split_comma_separated_str` in commit [c52d43d](https://github.com/Azure/azure-cli/pull/19478/commits/c52d43dc02ff1bacf7e1a6410f742e8c1c66fa43).